pkg/network/render.go

```diff
@@ -600,16 +601,30 @@ func renderAdditionalNetworks(conf *operv1.NetworkSpec, manifestDir string) ([]*
 	return out, nil
 }
 
+func getMultusAdmissionControllerReplicas(bootstrapResult *bootstrap.BootstrapResult) int {
+	replicas := 2
+	if bootstrapResult.Infra.ControlPlaneTopology == configv1.ExternalTopologyMode {
+		if bootstrapResult.Infra.InfrastructureTopology == configv1.SingleReplicaTopologyMode {
+			replicas = 1
+		}
+	} else if bootstrapResult.Infra.ControlPlaneTopology == configv1.SingleReplicaTopologyMode {
+		replicas = 1
+	}
```

Comment:
```suggestion
	if bootstrapResult.Infra.ControlPlaneTopology == configv1.SingleReplicaTopologyMode {
		replicas = 1
	}
```

Reply:
When control plane is external it gets scheduled on the workers, so we should look at the infra topology

https://github.com/openshift/cluster-network-operator//blob/3d24e5dfe62c67acd8243f16169ae0d19db5e3ff/bindata/network/multus-admission-controller/admission-controller.yaml#L82-L85

Reply:
:man_facepalming: 
I didn't notice the difference, thanks.